app/models/foreman_virt_who_configure/output_generator.rb

```diff
@@ -105,6 +105,7 @@ def virt_who_output(format = nil)
 rhsm_username=#{service_user_username}
 rhsm_encrypted_password=$user_password
 rhsm_prefix=/rhsm
+kubeconfig=#{config_path} if config.hypervisor_type == 'kubevirt'
```

Comment:
@ares Is this OK? How can I update UI to ask for `config_path`?

Reply:
this won't work, the if is now part of the string that starts with `<< EOS` on line 53, I'd define a variable above it, on line 52, something like `kubeconfig = config.hypervisor_type == 'kubevirt' ? "\nkubeconfig=#{config.kubeconfig_path} : ''` and then here, I'd just add `#{kubeconfig}` at after `/rhsm` on line above

you will also need to add new kubeconfig_path attribute to Config model, so add a migration that adds this new string field to the table, then update the form https://github.com/theforeman/foreman_virt_who_configure/blob/master/app/views/foreman_virt_who_configure/configs/steps/_connection_form.erb with new text_field, but also add JS to show it only for CNV (similar to username and password)

Reply:
if there's default path, make it also default value of that SQL column, that will autofill the form for new configurations

Reply:
Done